react/react-with-local-storage/react-with-local-storage_stored-note/src/App.jsx

```diff
@@ -1,11 +1,11 @@
-import { useState } from "react";
 import FontSelector from "./components/FontSelector";
 import Note from "./components/Note";
 import "./App.css";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function App() {
-  const [note, setNote] = useState("");
-  const [font, setFont] = useState("system-ui");
+  const [note, setNote] = useLocalStorageState("");
```

Comment:
believe you need to set the default useLocalStorageState("note", { defaultValue: "" } );